va/caa.go

```diff
@@ -25,10 +25,11 @@ func (va *ValidationAuthorityImpl) IsCAAValid(
 
 	if prob != nil {
 		typ := string(prob.Type)
+		detail := fmt.Sprintf("While processing CAA for %s : %s", *req.Domain, prob.Detail)
```

Comment:
Could you remove the space before ":"?

Reply:
Sure - https://github.com/letsencrypt/boulder/pull/3116/commits/739e64be2976b8784d28224582aa41e5329e512a